new/components/amqpinflux/scripts/create.sh

```diff
@@ -26,7 +28,18 @@ yum_install ${AMQPINFLUX_RPM_SOURCE_URL}
 
 create_service_user ${AMQPINFLUX_USER} ${AMQPINFLUX_HOME}
 
+if [[ "$(ctx node properties rabbitmq_ssl_enabled)" == 'True' ]]; then
```

Comment:
as per our convention, please assign this to a global variable.
Also, you can instead do `ctx -j` to get the boolean value.